fenix/app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -72,5 +72,5 @@ object FeatureFlags {
     /**
      * Allows users to enable Firefox Suggest.
      */
-    val fxSuggest = Config.channel.isDebug
+    val fxSuggest = Config.channel.isNightlyOrDebug
```

Comment:
Whoops, I think this was meant for #4049! 😅

Reply:
🤦‍♀️ been switching branches too often, losing track 😅